docs/getting-started/building-bpfman.md

```diff
@@ -32,8 +32,12 @@ Major kernel features leveraged by bpfman:
 * **BPF Perf Link:** Support BPF perf link for tracing programs (Tracepoint, Uprobe and Kprobe)
   which enables pinning for these program types.
   Introduced in Kernel 5.15.
+* **Relaxed CAP_BPF Requirement:** Prior to Kernel 5.19, all eBPF system calls required CAP_BPF.
+  This meant eBPF enabled applications which need to access their eBPF maps shared needed to have
+  the CAP_BPF Linux capability.
+  With the kernel 5.19 change, CAP_BPF is only required for load and unload requests.
```

Comment:
```suggestion
  This required userspace programs that wanted to access eBPF maps to have the CAP_BPF Linux capability.
```

Reply:
I think you wanted the previous sentence reworked.
```
-  This meant eBPF enabled applications which need to access their eBPF maps shared needed to have the CAP_BPF Linux capability.
+  This required userspace programs that wanted to access eBPF maps to have the CAP_BPF Linux capability.
```
That's what I changed.